core/src/main/java/org/apache/jsieve/comparators/AsciiCasemap.java

```diff
@@ -40,6 +40,8 @@ public AsciiCasemap() {
      * @see org.apache.jsieve.comparators.Equals#equals(String, String)
      */
     public boolean equals(String string1, String string2) {
+	if (string1 == null && string2 == null) return true;
```

Comment:
You have an indentation problem, this if is not at the same level than the other one.

We also tend to always use a block after a **if**.